packages/radio-group/src/Radio.tsx

```diff
@@ -295,16 +294,19 @@ export type RadioProps = Omit<React.ComponentPropsWithoutRef<'input'>, 'size'> &
 function Radio({
   children,
   className,
-  onChange,
+  onChange = () => {},
```

Comment:
Adding a default here because on the first render, there's no onChange prop and React complains that we're setting checked in the HTML without an onchange rendering it a `readonly` field, which is not true.